amgut/lib/data_access/test/test_ag_data_access.py

```diff
@@ -43,7 +29,20 @@ def test_deleteAGParticipantSurvey(self):
         raise NotImplementedError()
 
     def test_getConsent(self):
-        raise NotImplementedError()
+
+        res = self.ag_data.getConsent("8b2b45bb3390b585")
+        exp = {'date_signed': 'None',
+               'assent_obtainer': None,
+               'age_range': None,
+               'parent_1_name': 'REMOVED',
+               'participant_email': 'REMOVED',
+               'parent_2_name': 'REMOVED',
+               'ag_login_id': '000fc4cd-8fa4-db8b-e050-8a800c5d02b5',
+               'deceased_parent': 'false',
```

Comment:
can you create an issue on this? given that `is_juvenile` is actually boolean, this probably should be too